code/modules/mob/living/silicon/robot/dogborg/dog_sleeper.dm

```diff
@@ -0,0 +1,454 @@
+//Sleeper
```

Comment:
Gee, look. Isn't it 454 new shiny lines of Odysseus mounted sleeper code made into freaking vore fetish?